addons/velaux/resources/parameter.cue

```diff
@@ -15,4 +15,6 @@ parameter: {
 	serviceAccountName: *"kubevela-vela-core" | string
 	// +usage=Specify the service type.
 	serviceType: *"ClusterIP" | "NodePort" | "LoadBalancer" | "ExternalName"
+	// +usage=Specify the imagePullSecrets for private image registry, eg. " ["regcred"] "
+	imagePullSecrets?: string
```

Comment:
```suggestion
	imagePullSecrets?: [...string]
```

Reply:
I tried this, but failed, vela addon enable command raises error:
`Error: render addon application fail: fail to render cue tmpl`

As you know the workload of velaux is Component webservice, and its parameter type of imagePullSecrets is  [...string], but source code of vela cli
[func NewAddonEnableCommand(c common.Args, ioStream cmdutil.IOStreams) *cobra.Command {](https://github.com/oam-dev/kubevela/blob/7e102d70e0146669de97cfe04023bfd30f7c53d9/references/cli/addon.go#L111)

use function "parseToMap" to convert cli parameters to one map[string][string]

https://github.com/oam-dev/kubevela/blob/7e102d70e0146669de97cfe04023bfd30f7c53d9/references/cli/addon.go#L123

https://github.com/oam-dev/kubevela/blob/7e102d70e0146669de97cfe04023bfd30f7c53d9/references/cli/addon.go#L233

so can not use [...string] to receive here.

Reply:
/cc @wangyikewxgm

Reply:
In this case, we should support `[...string]`. It would be very complex if users have to use `" ["regcred"] "` like this.

Reply:
> In this case, we should support `[...string]`. It would be very complex if users have to use `" ["regcred"] "` like this.

Yes, I think [...string] is better solution, but it may need some efforts to parse  arguments to be "array" type for vela addon command as what I mentioned in previous comment. @wangyikewxgm any idea?

Reply:
ping @wangyikewxgm

Reply:
> > In this case, we should support `[...string]`. It would be very complex if users have to use `" ["regcred"] "` like this.
> 
> Yes, I think [...string] is better solution, but it may need some efforts to parse arguments to be "array" type for vela addon command as what I mentioned in previous comment. @wangyikewxgm any idea?

Yes vela cli need some change to support list parameter currently.